diff --git a/node_modules/sequelize/types/lib/model.d.ts b/node_modules/sequelize/types/lib/model.d.ts
index 6294c3e..2ec173f 100644
--- a/node_modules/sequelize/types/lib/model.d.ts
+++ b/node_modules/sequelize/types/lib/model.d.ts
@@ -1936,7 +1936,7 @@ export abstract class Model<T = any, T2 = any> extends Hooks {
   public static create<M extends Model>(
     this: { new (): M } & typeof Model,
     values?: object,
-    options?: CreateOptions
+    options?: CreateOptions & { returning: true }
   ): Promise<M>;
   public static create(values: object, options: CreateOptions & { returning: false }): Promise<void>;
 
